liquibase-core/src/main/resources/META-INF/native-image/liquibase/resource-config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "resources":[
+    {"pattern":"\\Qliquibase.build.properties\\E"},
+    {"pattern":"\\Qwww.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd\\E"}
```

Comment:
Why is there only the 3.1 XSD?

This should be the list of XSDs for > 3.x:

    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.2.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.7.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.10.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.0.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd",
    "www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd",
    "www.liquibase.org/xml/ns/pro/liquibase-pro-3.8.xsd",
    "www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd",
    "www.liquibase.org/xml/ns/pro/liquibase-pro-3.10.xsd",
    "www.liquibase.org/xml/ns/pro/liquibase-pro-4.0.xsd",
    "www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd",

Reply:
Thanks for the review @derkoe. As I mentioned in this PR I don't know the internals of Liquibase so this is one of the things that was missing. I only added that the 3.1 XSD because I only needed that to make Liquibase work with Micronaut.
Glad you took a look at this :-)

If you need any other change to consider merging this PR, please let me know.